src/qt/diagnosticsdialog.cpp

```diff
@@ -279,19 +281,31 @@ void DiagnosticsDialog::VerifyWalletIsSynced()
 {
     UpdateTestStatus(__func__, ui->verifyWalletIsSyncedResultLabel, pending, NA);
 
-    if (!OutOfSyncByAge())
+    if (!g_synced_before and OutOfSyncByAge())
```

Comment:
TIL but maybe `&&`?

Reply:
Ha... interesting that worked. I changed it to &&